src/blame_git.c

```diff
@@ -93,10 +93,10 @@ static bool same_suspect(git_blame__origin *a, git_blame__origin *b)
 }
 
 /* find the line number of the last line the target is suspected for */
-static int find_last_in_target(git_blame *blame, git_blame__origin *target)
+static int64_t find_last_in_target(git_blame *blame, git_blame__origin *target)
```

Comment:
Does this need to be signed or are we overloading the line number with the error code?  Why not:

    static int find_last_in_target(size_t *out, git_blame *blame, git_blame__origin *target)